solutions/standard/variables.tf

```diff
@@ -126,3 +135,114 @@ variable "skip_en_kms_auth_policy" {
   description = "Set to true to skip the creation of an IAM authorization policy that permits all Event Notification instances in the resource group to read the encryption key from the KMS instance."
   default     = false
 }
+
+########################################################################################################################
+# COS
+########################################################################################################################
+
+variable "cos_destination_name" {
+  type        = string
+  description = "The name to give the IBM Cloud Object Storage destination which will be created for storage of failed delivery events."
+  default     = "COS Destination"
+}
+
+variable "cos_bucket_name" {
+  type        = string
+  description = "The name of an existing IBM Cloud Object Storage bucket which will be used for storage of failed delivery events."
```

Comment:
description is wrong here - this is name that will be given to the COS bucket that this solution will created.

`"The name to use when creating the Cloud Object Storage bucket for storing failed events (NOTE: bucket names are globally unique). If 'add_bucket_name_suffix' is set to true, a random 4 characters will be added to this name to help ensure bucket name is globally unique."`